guides/common/modules/proc_creating-new-ssh-keys-for-remote-execution.adoc

```diff
@@ -0,0 +1,19 @@
+[id="creating-new-ssh-keys-for-remote-execution_{context}"]
+= Creating new SSH keys for remote execution
+
+By default, {Project} uses a non-passphrase protected SSH key to execute remote jobs on hosts.
+You can create additional passphrase protected or non-passphrase protected SSH keys to authenticate for remote execution jobs on hosts.
```

Comment:
The word `additional` could be misleading in this context. It makes it read as if you could use multiple keys at once, while a smart proxy expects exactly one key.